library/Exceptions/ValidationException.php

```diff
@@ -47,6 +47,28 @@ public static function stringify($value)
         }
     }
 
+    /**
+     * @param $value
+     * @return string
+     */
+    public static function stringifyArray($value)
+    {
+        $items = array();
+        foreach ($value as $val) {
+            if (is_object($val)) {
+                $items[] = self::stringifyObject($val);
+            } elseif (is_string($val)) {
+                $items[] = "'$val'";
+            } else {
+                $items[] = (string) $val;
```

Comment:
Maybe : 

```php 
$array = [    
    1,
    "respect",
    [1,2 , [ "respect" ]]
]
```
could be printed as 

```
(1, "respect", Array) 
```
or we use recursion here with some way to define depth of recursion.